app/marketing/mails.py

```diff
@@ -35,10 +37,14 @@
     render_start_work_approved, render_start_work_new_applicant, render_start_work_rejected, render_tip_email,
 )
 from sendgrid.helpers.mail import Content, Email, Mail, Personalization
+from sendgrid.helpers.stats import Category
+
 
+def func_name():
+    return inspect.stack()[0][3]
 
 def send_mail(from_email, _to_email, subject, body, html=False,
-              from_name="Gitcoin.co", cc_emails=None):
+              from_name="Gitcoin.co", cc_emails=None, categories=['default']):
```

Comment:
We should default to `categories=None` and `if categories is None: categories = ['default']`